src/main/java/org/jabref/logic/util/FileUtil.java

```diff
@@ -308,26 +326,96 @@ public static String createDirNameFromPattern(BibDatabase database, BibEntry ent
      * @param rootDirectory the rootDirectory that will be searched
      * @return the path to the first file that matches the defined conditions
      */
-    public static Optional<Path> find(String filename, Path rootDirectory) {
+    public static Optional<Path> findSingleOld(String filename, Path rootDirectory) {
```

Comment:
So if this is deprecated, add `@deprecated`?

Reply:
I don't think its deprecated, will give it another name